### PR TITLE
Break out configurations into separate classes

### PR DESCRIPTION
**Pull request by jhunzik**

### CODE REVIEW

**Comment by jhunzik** [comment]
build now

File: multi-int-store/src/main/java/com/connexta/multiintstore/config/ManagerConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
```
Comment (ahoffer) [inline multi-int-store/src/main/java/com/connexta/multiintstore/config/ManagerConfiguration.java]: ManagerConfiguration is vague. How about "StorageManagerConfiguration" or "StorageConfiguration"?

Comment (jhunzik) [reply multi-int-store/src/main/java/com/connexta/multiintstore/config/ManagerConfiguration.java]: I like StorageManagerConfiguration

**Comment by jhunzik** [comment]
build now

**Comment by jhunzik** [comment]
rebased. build now

**Comment by jhunzik** [comment]
aborted. build now

**Comment by jhunzik** [comment]
build now. I just ran a successful full build locally, so I don't think it's our code. Giving it one more go.

**Comment by jhunzik** [comment]
✅ Hero Successful. I was able to start the Ingest and MIS services with all of their beans and upload products as expected.

**Pull request merged**